sources/Component/Movable/MovableComponent.hpp

```diff
@@ -15,9 +15,10 @@ namespace BBM
 	private:
 		//! @brief The acceleration of this entity.
 		Vector3f _acceleration;
+	public:
 		//! @brief The velocity of the entity.
 		Vector3f _velocity;
```

Comment:
Add a reset method but the velocity is overriden at each frame, it will lead devs into errors.